.changeset/tame-knives-fold.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cambly/syntax-floating-components": patch
+"@cambly/syntax-core": patch
+---
+
+Add individual links to components
```

Comment:
Aside, newb question:
👀 ok this file has me curious.  what system is using this and why is it titled `tame-knives-fold`? 👀

Reply:
@benjroy See https://github.com/changesets/changesets - that name gets auto generated.

https://github.com/changesets/changesets/blob/main/docs/common-questions.md#each-changeset-is-its-own-file

> We use random human readable names by default for these files to avoid collisions when generating them, but there's no harm that will come from renaming them.